src/starbunk/bots/strategy-loader.ts

```
@@ -24,27 +27,68 @@ interface StrategyLoaderConfig {
 }
 
 // Export the class for testing purposes
-export class StrategyBotLoader {
+export class ReplyBotLoader {
+	public static async loadBots(): Promise<ReplyBot[]> {
+		logger.info('Loading reply bots...');
+		const loadedBots: ReplyBot[] = [];
+
+		try {
+			// Initialize CovaBot first
+			await this.initializeCovaBot();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)